docs/site/lib/create-metadata.ts

```diff
@@ -60,15 +58,7 @@ export const createMetadata = ({
   }
 
   const formatTitle = (): string => {
-    if (canonicalPath === "/repo") {
-      return "Turborepo";
-    }
-
-    if (product === "repo") {
-      return `${title} | Turborepo`;
-    }
```

Comment:
Maybe worth keeping the | Turborepo in the title for tabs so you can quickly find your Turborepo docs

Reply:
Oops, good catch.